sbc_translation/models/translation_user.py

```diff
@@ -31,8 +31,9 @@ class TranslationUser(models.Model):
         "res.lang.compassion", domain=[("translatable", "=", True)],
         help="Utility field only used for the search view"
     )
-    search_competence_id = fields.Many2one(
-        "translation.competence", help="Utility field only used for the search view"
+    search_competence_ids = fields.Many2one(
```

Comment:
I don't see the point of renaming the field, especially if you put a suffix that don't reflect the correct field type.